CommunityToolkit.Net.Authentication/CommunityToolkit.Net.Authentication.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Title>Community Toolkit .NET Standard Auth Services</Title>
+    <Description>
+        This package includes .NET Standard authentication helpers such as:
+        - BaseProvider:
+        - IProvider:
+        - MockPRovider:
+        - MsalProvider:
+        - ProviderManager:
+        - ProviderManagerChangedState:
+        - ProviderState:
+        - ProviderStateChangedEventArgs:
+        - ProviderUpdatedEventArgs:
+    </Description>
+    <PackageTags>Community Toolkit Provider Authentication Auth</PackageTags>
+
+    <!-- This is a temporary workaround for https://github.com/dotnet/sdk/issues/955 -->
+    <DebugType>Full</DebugType>
+    <Configurations>Debug;Release;CI</Configurations>
+    <Platforms>AnyCPU;ARM;ARM64;x64;x86</Platforms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Graph.Auth" Version="1.0.0-preview.6" />
```

Comment:
Ah, so we're still relying on this package... hmmm. This is what's pulling in MSAL/ (So if you only want the Windows provider, you're still pulling in MSAL?)

Also, this is the part that's getting rolled into the new Graph SDK which includes everything together... @nmetulev thoughts on if we should try and isolate that for the future vs. knowing eventually we'll pull in the whole Graph SDK... This seems like it's going to be a big problem for consumption of their v2 as we had mentioned.

What pieces are we still referencing from here (besides MSAL)? Like should we have an explicit `CommunityToolkit.Net.Authentication.Msal` or something?

I wonder if we investigate the source only style package, so rather than having another DLL for just the single MSAL Provider, it effectively just includes the CS file and it builds as part of the consuming app... 🤔 @Sergio0694 thoughts here? Is this [the article you've been looking at for this 'feature'](https://medium.com/@attilah/source-code-only-nuget-packages-8f34a8fb4738)?

Reply:
I've moved MsalProvider to it's own package, `CommunityToolkit.Net.Authentication.Msal`, and it's now based on the official `Microsoft.Identity.Client.Extensions.Msal` library. 

I looked into the source code only package, but it has an unfortunate side effect of not being able to use project references anymore. Which means I can't reference the msal package from the sample app, because it emits a nuget from build, not a DLL. I think it's probably fine to leave it as a package, imo.